ironfish/src/rpc/clients/tlsClient.ts

```diff
@@ -23,9 +23,7 @@ export class RpcTlsClient extends RpcTcpClient {
         client.off('secureConnection', onSecureConnect)
         client.off('error', onError)
 
-        if (ErrorUtils.isConnectRefusedError(error)) {
-          reject(new RpcConnectionRefusedError())
-        } else if (ErrorUtils.isNoEntityError(error)) {
+        if (ErrorUtils.isConnectFailError(error)) {
           reject(new RpcConnectionRefusedError())
```

Comment:
We have other error types defined alongside `RpcConnectionRefusedError` that might be more appropriate depending on the error.

For example, if `isConnectTimeOutError` or `isConnectResetError`, then it might be better to use [RpcConnectionLostError](https://github.com/iron-fish/ironfish/blob/02939ae309a98205781908fd31e218aee4e62fea/ironfish/src/rpc/clients/errors.ts#L33)

Reply:
Updated.